specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/Global.json

```diff
@@ -0,0 +1,329 @@
+        "operationId": "Global_GetSubscriptionOperations",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "description": "Location name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ArmOperationResponse"
+              }
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets all the operations for a subscription in a given region": {
+            "$ref": "./examples/GetSubscriptionOperations.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "Global"
+        ],
+        "summary": "Gets an operation in a subscription and given region",
+        "description": "Description for Gets an operation in a subscription and given region",
+        "operationId": "Global_GetSubscriptionOperation",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "description": "Location name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "description": "Operation Id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ArmOperationResponse"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets an operation in a subscription and given region": {
+            "$ref": "./examples/GetSubscriptionOperation.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/operations/{operationId}": {
```

Comment:
>subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/operations/{operationId} [](start = 6, length = 100)

Why do you need this given the above api can track an async operation?

Reply:
removed.